src/mmw/js/src/core/views.js

```diff
@@ -164,11 +166,15 @@ var MapView = Marionette.ItemView.extend({
         this._areaOfInterestLayer = areaOfInterestLayer;
         this._modificationsLayer = modificationsLayer;
 
+        this.model.set('disable_geolocation', false);
```

Comment:
Consider renaming the attribute so the behavior is not a double negative. For example, `geolocation_enabled`, which would be easier to read in the `if` condition below. `if (geolocation_enabled)` is easier to read than `if(!disable_geolocation)`.

Also consider moving the initialization to the `defaults` in `models.js`.